rasa/core/training/interactive.py

```diff
@@ -1130,9 +1136,23 @@ async def _validate_action(
 
     Returns `True` if the prediction is correct, `False` otherwise."""
 
-    question = questionary.confirm(f"The bot wants to run '{action_name}', correct?")
+    if action_name == ACTION_UNLIKELY_INTENT_NAME:
+        question = questionary.confirm(
+            f"The bot wants to run '{action_name}' "
+            f"to indicate that the last user message was unexpected "
+            f"at this point in the conversation. "
+            f"Check out IntentTEDPolicy ({DOCS_URL_POLICIES}/#intent-ted-policy) "
+            f"to learn more."
```

Comment:
Do we need to ask the user here to press Y/N to continue?

Reply:
That's what we've decided to do, yes (to keep it consistent with the rest)

Reply:
Ah ok, then maybe it should be explicitly stated? Unless `questionary` takes care of this 😄

Reply:
IMO this should instead be written in docs with the explanation what `action_unlikely_intent` is and how it works. We can also write it here but I am afraid the message will be too long in this case.

Reply:
Ah I meant maybe it's not obvious what keys to press from the question and that should be explicitly stated, thats all 👍

Reply:
Oh, got it! Actually, `questionary` handles that and shows the available options automatically :)